install/index.adoc

```diff
@@ -178,6 +178,18 @@ one HAProxy load balancer, and two
 xref:../architecture/infrastructure_components/kubernetes_infrastructure.adoc#node[nodes]
 using the `native` HA method. etcd runs as static pods on the master nodes:
 
+[IMPORTANT]
+====
+Routers and master nodes must be load balanced to have a highly available and
+fault-tolerant environment. Red Hat recommends the use of an enterprise-grade
+external load balancer for production environments. This Load balancing applies
+to the masters, nodes, and hosts running the {product-title} routers.
+Transmission Control Protocol (TCP) layer 3/4 load balancing, in which the load
```

Comment:
We may just want to say layer 4 

@sheriff-rh can you comment here if layer 3/4 is correct (my networking, terminology is rusty).

Reply:
Correct @sferich888 TCP is layer 4.